index.html

```diff
@@ -15,82 +16,78 @@ <h1>Steganography Online</h1>
     </ul>
 
     <div id="my-tab-content" class="tab-content">
-      <div class="tab-pane active" id="encode">
-        <div>
-          <h2>Encode message</h2>
-          <p class="alert alert-info">
-            To encode a message into an image, choose the image you want to use, enter your text and hit the <strong>Encode</strong> button. <br/>
-            Save the last image, it will contain your hidden message. <br/>
-            Remember, the more text you want to hide, the larger the image has to be. In case you chose an image that is too small to hold your message you will be informed.<br/><br/>
-            Neither the image nor the message you hide will be at any moment transmitted over the web, all the magic happens within your browser.
-          </p>
-          <form class="form">
-            <div class="form-group">
-              <input class="form-control" type="file" name="baseFile" onchange="previewEncodeImage()">
-            </div>
-
-            <div class="form-group">
-              <textarea class="form-control message" rows="3" placeholder="Enter your message here"></textarea>
-            </div>
-
-            <div class="form-group">
-              <button class="encode btn btn-default pull-right" onclick="encodeMessage()">Encode</button>
-            </div>
-          </form>
-          <div class="clearfix"></div>
+  <div class="tab-pane active" id="encode">
```

Comment:
Please keep the indentation.

Reply:
Sorry for messing up the indentation. Thanks for the quick response and for building this tool. It assisted me in a CTF challenge. I've corrected the indentation.